.travis.yml

```diff
@@ -48,13 +48,11 @@ matrix:
    - compiler: gcc
      env: COVERITY=1
      os: linux
-     dist: trusty
```

Comment:
I'd rather prefer if we'd delete the global "dist" key. It's nice to have all that information directly in the build matrix.

Reply:
`dist` is a [global](https://docs.travis-ci.com/user/reference/trusty/) key though, so I can't really use the other. Else I'd have kept the other one 😉.

Reply:
I don't think this is true. For some time, our matrix contained jobs for both Trusty and Precise, and it worked perfectly fine